src/data/causality.graphql

```diff
@@ -236,7 +236,7 @@ type Query {
   ): LotStandingConnection!
 
   "Returns an increment policy with specific `id`."
-  incrementPolicy(id: String!): IncrementPolicy
+  incrementPolicy(id: ID!): IncrementPolicy
```

Comment:
These are not showing up in the resulting MP schema updates above. I think it's because they are [not permitted](https://github.com/artsy/metaphysics/blob/b1a67f53608f29777009006f5d1ca98d1d6054f5/src/lib/stitching/causality/schema.ts#L15) when merging schemas. But let me know if I am wrong.